package.json

```diff
@@ -34,6 +34,7 @@
     "migrate-down": "migrate down",
     "migrate": "migrate up",
     "postshrinkwrap": "node -e \"fs.utimesSync('./node_modules', new Date(), new Date());\"",
+    "reformat-files": "prettier --ignore-path .eslintignore --write",
```

Comment:
It asks for files or globs — so e.g. `npm run reformat-files modules/**/api/*`

There might be something else off since my last commit if it doesn't work standalone. 🤔

Reply:
Well, I haven't tested with the glob parameter. Could that be the case?

Reply:
Should work with single file part too.

Reply:
> Should work with single file path too.

It does.

The remaining issue is: it still expects the `/** @format */`, otherwise nothing gets formatted.